packages/rspack-test-tools/tests/configCases/builtins/html-issue-8410/index.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs");
+const path = require("path");
+
+it("html plugin should respect output.publicPath", () => {
+	const htmlPath = path.join(__dirname, "./index.html");
+	const htmlContent = fs.readFileSync(htmlPath, "utf-8");
+	console.log(htmlContent);
```

Comment:
LGTM, should remove this `console.log` and then it can be merged

Reply:
done